tests/resources/test_multipartycalls.py

```diff
@@ -229,7 +229,7 @@ def test_add_participant(self):
     @with_response(200)
     def test_start_MPC(self):
 
-        request_body = {'status': 'active'}
```

Comment:
Need to change, because i am creating an localObject of type array.
File - plivo/resources/multipartycall.py line 379